src/CameraDevice.h

```diff
@@ -42,6 +42,7 @@ class CameraDevice {
         STATE_ACTIVE = 1,
     };
 
+    virtual std::string getDevice() = 0;
```

Comment:
Method `CameraComponent::getDevice()` doesn't intuitively tell whether it returns device name.
I prefer to call `CameraComponent::getDeviceName()`

Reply:
Fixed, its not name but Id